src/libAtomVM/module.h

```diff
@@ -262,6 +283,71 @@ static inline const uint8_t *module_get_str(Module *mod, size_t offset, size_t *
     return ((const uint8_t *) mod->str_table) + 8 + offset;
 }
 
+/*
+ * @brief Get the function name and arity of a function defined at a label.
+ *
+ * @details This function is used to get the function name and arity from the
+ * label -> fun/arity table that is built during code loading.
+ * See module_set_label_fun_arity.
+
+ * @param mod the module
+ * @param label the current label used to look up the function/arity
+ * @param function_name (output) the function name, as an AtomString.
+ * @param arity (output) the function arity
+ */
+void module_get_fun_arity(Module *mod, int label, AtomString *function_name, int *arity);
```

Comment:
This function can be implemented differently using LocT and ExpT tables, and can work with any label (not necessarily the first function label) by returning the function with the highest label which is less or equal than the provided label.